app/main.py

```diff
@@ -1,3 +1,65 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+
+        if isinstance(other, int) or isinstance(other, float):
+            self.km += other
+            return self
```

Comment:
no need to use isinstance twice, u can just use `else` here
also instead of double isinstance u can write like this `isinstance(other, (int, float))`
also u are returning the same thing for 2 times, could u use only one return here?

Reply:
fix this everywhere